packages/extension-chakra-store-locator/package.json

```diff
@@ -41,7 +41,6 @@
   "peerDependencies": {
     "@chakra-ui/react": "^2.8.2",
     "@loadable/component": "^5.15.3",
-    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
```

Comment:
I will do a major version bump preview release for commerce-sdk-react @ `4.0.0-preview.0`

Reply:
resolved